tests/tools/collab/conftest.py

```
@@ -0,0 +1,925 @@
+            protocol_version = 'HTTP/1.1'
+
+            def _answer(self):
+                self.send_response(status)
+                self.send_header('Content-Type', 'application/json')
+                self.send_header('Content-Length', str(len(body)))
+                self.end_headers()
+                self.wfile.write(body)
+
+            do_GET = do_POST = do_PUT = do_HEAD = _answer  # noqa: N815
+
+            def log_message(self, format, *args):
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)